uber_sante/utils/dbutil.py

```diff
@@ -3,15 +3,21 @@
 
 class DBUtil:
 
-    def __init__(self, app, test):
-        self.app = app
-        self.test = test
-        # if test:
-        #     self.init_test_database()
-        #     self.connect_db = self.connect_test_database
-        # else:
-        self.init_database()
-        self.connect_db = self.connect_database
+    instance = None
+    app = None
+
+    def __init__(self, app, test=False):
+        if DBUtil.instance is None:
+            DBUtil.instance = self
+            self.app = app
+            self.test = test
```

Comment:
As i have previously stated, you have decoupled this class from the test variable, so why are we keeping it at the instance level.

Reply:
Another good find